manylinux/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM quay.io/pypa/manylinux2010_x86_64
+
+# yum provides only an old version of Eigen3 (too old)
+RUN cd /tmp && \
+    curl https://bitbucket.org/eigen/eigen/get/3.3.7.tar.bz2 | tar -jx && \
```

Comment:
Are there any licensing concerns here ? I see they have an option to enforce only MPL2 and BSD code in the install - or do we use features outside of that stripped down set.

Reply:
Thanks, I wasn't aware of that. I'll investigate. We're using only very basic features so I imagine it won't be an issue.

Reply:
Reading the MPL2 FAQ it looks like we need to inform recipients where they can receive the source code for Eigen. I'll add it to the LICENSE file, unless you know of a better place to put it.